cost-analyzer/values-openshift.yaml

```diff
@@ -0,0 +1,36 @@
+kubecostProductConfigs:
+  clusterName: CLUSTER_NAME
+  projectID: CLUSTER_NAME
+
+prometheus:
+  nodeExporter:
+    enabled: false
+  serviceAccounts:
+    nodeExporter:
+      create: false
+  kube-state-metrics:
```

Comment:
Bolt suggested doing both on the other PR:
  kubeStateMetrics:
    enabled: false
  kube-state-metrics:
    disabled: true